tests/testdata/buf.build/twitchtv/twirp/v8.1.3/eliza/plugin.sum

```diff
@@ -0,0 +1 @@
+h1:ADqSwh7gUVVJH1yyFHYg5A1NpSmxGzc1/VbN9Pjh+5c=
```

Comment:
I think the CI build is failing due to recent changes where it builds with the separate_package option from registry opts (to test more scenarios).